src/fw/drivers/wscript_build

```diff
@@ -465,14 +465,15 @@ if bld.is_silk() and not bld.env.QEMU:
     )
 
 if bld.is_asterix() or bld.is_obelix():
-    # FIXME(ASTERIX,OBELIX): provide working implementation
```

Comment:
note: since this branch currently applies to obelix, you'll need to modify its board as well for things to compile. I'd suggest to split into asterix, leave obelix with stubs for now.